impl/src/main/java/org/jboss/weld/bootstrap/events/configurator/ObserverMethodConfiguratorImpl.java

```diff
@@ -330,6 +330,14 @@ public boolean isEventMetadataRequired() {
            return true;
        }
 
-   }
+       @Override
+       public String toString() {
+           return "Configurator observer method [Bean class = " + getBeanClass().toString()
```

Comment:
I'm not sure why but we allow to set `null` bean class. If you remove `toString()` here, then this method should not throw NPE and the resulting string should be the same.

Reply:
Nice catch!